src/app/pages/component-viewer/component-api.html

```diff
@@ -1,5 +1,5 @@
 <ng-container *ngIf="componentViewer.componentDocItem | async; let docItem">
-  <span class="cdk-visually-hidden" tabindex="-1" #initialFocusTarget>
+  <span class="cdk-visually-hidden" tabindex="-1" #initialFocusTarget focusOnNavigation>
```

Comment:
Can we remove this and use `@ViewChild(FocusOnNavigation)` instead in `ComponentBaseView`?

Reply:
I'm not sure if this is what you meant but  I removed the focusOnNavigation directives on the api/over/example pages and now just have 1 on the base component viewer

Reply:
Oh, that's a good change as well! What I meant was to remove the `#initialFocusTarget` and change `@ViewChild('initialFocusTarget') focusTarget: ElementRef;` in `ComponentBaseView` to `@ViewChild(FocusOnNavigation, {read: ElementRef}) focusTarget: ElementRef;` since we have a directive now that we can use as the selector and don't really need the 'initialFocusTarget' ref. It's not really necessary though, just a nit.